modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequestForm.testcase

```diff
@@ -82,21 +85,23 @@ definition {
 	@description = "LPS-162902 - Verify that can create a Request by clicking on 'Create a Request'"
 	@priority = 4
 	test CanCreateRequestForm {
-		task ("When The employee user creates a request") {
+		task ("Given that the Employee user create a new Request") {
```

Comment:
Please change this task to : `"Given that the Employee user creates a new Request"`